.travis.yml

```diff
@@ -14,9 +14,6 @@ services:
 
 # list any PHP version you want to test against
 php:
-  - 7.1.3
-  - 7.1.4
```

Comment:
Snipe-IT v5 should work with PHP 7.1.3 or better.

Reply:
Ooops, my bad, i commented this for testing because travis testing didnt need those (was broken completely) - will enable it again